test/parse/test-doc-whitespace.vows.js

```diff
@@ -9,16 +12,16 @@ wows.describe('errorHandle').addBatch({
 		assert(new DOMParser().parseFromString('<foo')+'', '<foo/>');
 	},
 	'document source':function(){
-		var testSource = '<?xml version="1.0"?>\n<!--test-->\n<xml/>'
-		var dom = new DOMParser().parseFromString(testSource,'text/xml')
-		assert(new XMLSerializer().serializeToString(dom), testSource)
+		var testSource = '<?xml version="1.0"?>\n<!--test-->\n<xml/>';
+		var dom = new DOMParser().parseFromString(testSource,'text/xml');
+		assert.strictEqual(new XMLSerializer().serializeToString(dom), testSource);
 	},
 	'test':function(){
 		var description = "<p>populaciji (< 0.1%), te se</p>";
 		var doc = new DOMParser().parseFromString(description, 'text/html');
-		assert(doc.toString(), '<p xmlns="http://www.w3.org/1999/xhtml">populaciji (&lt; 0.1%), te se</p>')
-	}
-}).export(module)
+		assert.strictEqual(doc.toString(), '<p xmlns="http://www.w3.org/1999/xhtml">populaciji (&lt; 0.1%), te se</p>');
+	},
+}).export(module);
```

Comment:
I think these should be reverted for now:

```suggestion
	}
}).export(module)
```